swayipc/meson.build

```diff
@@ -0,0 +1,8 @@
+lib_sway_ipc = static_library(
```

Comment:
I'd name the IPC library just "ipc" (instead of "swayipc") or `common/ipc`, since sway* directories contain executables.